### ISSUE TITLE
Building doc directory - mapbox_token needed

### ISSUE THREAD

**Issue by magic-lantern** [plotly/plotly.py#3483]
I'm wanting to build a version of plotly for offline use - specifically the documentation.

When I follow the very simple directions provided at https://github.com/plotly/plotly.py/tree/master/doc I do the following:

1.	git clone git@github.com:plotly/plotly.py.git
1.	cd plotly.py/doc
1.	conda create -n plotly_docs python=3.8 pip
1.	conda activate plotly_docs
1.	pip install -r requirements.txt
1.	make

I get the following error:

    make: *** No rule to make target 'python/.mapbox_token', needed by 'build/ipynb/.mapbox_token'.  Stop.

I'm not able to find information about mapbox_token.

**Comment by magic-lantern** [comment]
Clarifying my comment - perhaps this should be a separate issue?

I'm most interested in an older version of plotly python library. 

In the github version, removing the mapbox section from Makefile:

```make
$(IPYNB_DIR)/.mapbox_token: $(MD_DIR)/.mapbox_token
	@mkdir -p $(IPYNB_DIR)
	@echo "[symlink]    .mapbox_token"
	@cd $(IPYNB_DIR) && ln -s ../../$<

$(IPYNB_FILES): $(IPYNB_DIR)/.mapbox_token
```

Appears to allow everything to build correctly.

However in older versions If I comment out/remove the mapbox_token section it will start the build, but dies almost immediately with the error `TypeError: 'coroutine' object is not subscriptable`

 I'm not sure if this is due to mapbox portions not working or other error that is different. I suspect that the problem is due to some version mismatch of other software (Jupyter, Python, etc.)

**Comment by nicolaskruchten** [comment]
The contents of .mapbox_token should just be a valid mapbox.com token. It's just a matter of signing up and getting a free token :)

**Comment by magic-lantern** [comment]
Thanks @nicolaskruchten !

I created a free account, then copied my "Default public token" to the file `<gitrepo>/doc/python/.mapbox_token` then ran `make` 

While the build is still running, it seems to be working so far.

As this step seems to be missing from the docs (or at least is not in the README.md) I've created a fork and made a minor documentation update pull request. Will push shortly.
